proposals/atmospheric_correction.json

```diff
@@ -0,0 +1,67 @@
+{
+    "id": "atmospheric_correction",
+    "summary": "Apply atmospheric correction",
+    "description": "Applies an atmospheric correction that converts top of atmosphere reflectance values into bottom of atmosphere/top of canopy reflectance values.\nNote that multiple atmospheric methods exist, but may not be supported by all back-ends. The method parameter gives you the option of requiring a specific method, but this may result in an error if the back-end does not support it.",
+    "categories": [
+        "cubes",
+        "optical"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "description": "Data cube containing multi-spectral optical top of atmosphere reflectances to be corrected.",
+            "name": "data",
+            "schema": {
+                "subtype": "raster-cube",
+                "type": "object"
+            }
+        },
+        {
+            "description": "The atmospheric correction method to use.\n\nThe supported algorithms can vary between back-ends. Set to `null` (the default) to allow the back-end to choose, which will improve portability, but reduce reproducibility..",
+            "name": "method",
+            "optional": true,
+            "default": null,
+            "schema": {
+                "type": [
+                    "string",
+                    "null"
+                ],
+                "enum": [
+                    "FORCE",
+                    "iCor"
+                ]
+            }
+        },
+        {
+            "description": "The name of the digital elevation model to use, leave empty to allow the back-end to make a suitable choice. Not all back-ends may offer all digital elevation models.",
```

Comment:
The spec of this parameter is different to the one in sar_backscatter. I'd propose to align them. Also the note that "Not all back-ends may offer all digital elevation models" should not be in here. If a back-end doesn't support DEMs then the parameter should be removed or so by the implementinf back-end. But just the generic info is confusing.

Reply:
Align with sar_backscatter
Keep a string, backend can specialize to make it an enum or collection-id or union of both. (Depending on specific backend.)

Reply:
Default schema can use collection id by default?

Reply:
Done.